Makefile

```diff
@@ -103,6 +103,9 @@ draw-deps:
 	go get github.com/RobotsAndPencils/goviz
 	@goviz -i ./cmd/gaiad -d 2 | dot -Tpng -o dependency-graph.png
 
+update_gaia_lite_docs:
```

Comment:
Very very minor thing: I tried to adopt `-` separated target naming convention for the new Gaia makefile. Not a big deal either way

Reply:
Yes, copy-pasta and I agree!